assets/app/index.html

```diff
@@ -65,7 +65,9 @@
               <li class="divider"></li>
               -->
               <li>
-                <a href="logout">Log out</a>
+                <!-- Stop event propagation to avoid a runtime error when the
+                     dropdown tries to close itself as we're navigating away. -->
+                <a href="logout" ng-click="$event.stopPropagation()">Log out</a>
```

Comment:
The error definitely prevents some code from executing in the logout
controller. I'm not sure how important that logic is.

On Sunday, October 11, 2015, Jessica Forrester <notifications@github.com>
wrote:

> In assets/app/index.html
> <https://github.com/openshift/origin/pull/4869#discussion_r41714280>:
>
> > @@ -65,7 +65,9 @@
> >                <li class="divider"></li>
> >                -->
> >                <li>
> > -                <a href="logout">Log out</a>
> > +                <!-- Stop event propagation to avoid a runtime error when the
> > +                     dropdown tries to close itself as we're navigating away. -->
> > +                <a href="logout" ng-click="$event.stopPropagation()">Log out</a>
>
> the error doesn't prevent you from going to logout does it? wondering if
> we should just put in the ng-cloak stuff and leave the issue open as a P3
> to get a fix into bootstrap -> patternfly
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4869/files#r41714280>.
>